### PR TITLE
Weld tip - logging

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: news/2016-10-01-tip1-logging.asciidoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+On the other hand, this may cause problems when looking for a bug.
+There is an application class provided by a developer but what does the CDI container actually see?
+Well, it depends... because portable extensions are allowed to redefine this.
+And that's why Weld logs all the _"modification"_ operations performed by extensions, such as `ProcessAnnotatedType.veto()`:
+
+----
+DEBUG [Bootstrap] WELD-000148: ProcessAnnotatedType.veto() called by com.foo.MyExtension@50fed5b1 for [BackedAnnotatedType] public class com.foo.Foo
+----
+
+TIP: If your application deploys successfully, you can use link:http://docs.jboss.org/weld/reference/latest/en-US/html/devmode.html#devtools[Development Tools] to inspect the runtime info (more to be discussed in one of the next articles).
```
Comment (manovotn) [inline news/2016-10-01-tip1-logging.asciidoc]: `(more to be discussed in one of the next articles)` -> `(will be covered in one of the follow-up articles)`

File: news/2016-10-01-tip1-logging.asciidoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+* and others (see also link:https://github.com/weld/core/blob/master/impl/src/main/java/org/jboss/weld/logging/Category.java[org.jboss.weld.logging.Category, window="_blank"])
+
+So for example, if you're only interested in log messages related to *EL resolution* set the log level for the `org.jboss.weld.El` category to `TRACE` (or level with corresponding priority).
+
+[[bootstrap]]
+== Monitoring bootstrap
+
+CDI is designed to fail fast.
+An extensive validation is performed during application initialization in order to avoid malicious errors at runtime.
+Container usually also log a lot of interesting info during bootstrap.
```
Comment (manovotn) [inline news/2016-10-01-tip1-logging.asciidoc]: `log` -> `logs`

File: news/2016-10-01-tip1-logging.asciidoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+== Motivation
+
+Logging is usually no fancy stuff.
+But very often it's _the best thing to start with if problems arise_.
+Moreover, it's the easiest form of debugging.
+
+In this article you will find:
+
+* <<under_the_hood>> - how to control and configure the output
+* <<filter>> - how to filter the ouput
```
Comment (manovotn) [inline news/2016-10-01-tip1-logging.asciidoc]: How about this:
`<<filter>> - keeping logs small and to the point`

File: news/2016-10-01-tip1-logging.asciidoc [hunk 4]
```diff
@@ -63,10 +63,18 @@ Weld log messages are grouped into several categories:
...
 So for example, if you're only interested in log messages related to *EL resolution* set the log level for the `org.jboss.weld.El` category to `TRACE` (or level with corresponding priority).
+Another useful example is the check of link:http://docs.jboss.org/weld/reference/latest/en-US/html/configure.html#_weld_configuration[Weld configuration, window="_blank"].
+Simply set the log level to `DEBUG` and filter the `org.jboss.weld.Configuration` category:
+
+----
+DEBUG [Configuration] WELD-001907: Reading properties file: test.war/WEB-INF/classes/weld.properties
+DEBUG [Configuration] WELD-001903: Configuration key RESOLUTION_CACHE_SIZE already set to 1 000 in a source with higher priority, value 500 from system properties is ignored
+DEBUG [Configuration] WELD-001904: Unsupported configuration key found and ignored: com.foo.bar
+DEBUG [Configuration] WELD-001902: Configuration initialized: {CONCURRENT_DEPLOYMENT=false, RESOLUTION_CACHE_SIZE=1000, RELAXED_CONSTRUCTION=true}
+----
```
Comment (manovotn) [inline news/2016-10-01-tip1-logging.asciidoc]: Yes, that's what I meant, +1

File: news/2016-10-01-tip1-logging.asciidoc [hunk 5]
```diff
@@ -63,10 +63,18 @@ Weld log messages are grouped into several categories:
 * `org.jboss.weld.Bean`
 * `org.jboss.weld.Context`
 * `org.jboss.weld.El`
-* `org.jboss.weld.Conversation`
```
Comment (manovotn) [inline news/2016-10-01-tip1-logging.asciidoc]: One last thing, what did you remove `org.jboss.weld.Conversation`?
To make it shorter? Or did you mean to replace it with the Config one?

Comment (mkouba) [inline news/2016-10-01-tip1-logging.asciidoc]: Just to make it shorter...

**Comment by mkouba** [comment]
Merged.

**Pull request closed without merging**